includes/Core/Conversion_Tracking/Conversion_Event_Providers/WooCommerce.php

```diff
@@ -34,7 +33,7 @@ class WooCommerce extends Conversion_Events_Provider {
 	 * @return bool True if WooCommerce is active, false otherwise.
 	 */
 	public function is_active() {
-		return Plugin_Status::PLUGIN_STATUS_ACTIVE === Plugin_Status::get_plugin_status( 'woocommerce/woocommerce.php', 'https://woocommerce.com/' );
+		return defined( 'WC_PLUGIN_FILE' );
```

Comment:
For specific checks like this, and code that is written to directly integrate with another plugin, we can use much more efficient methods like this which are also more accurate than checking an option or presence on the file system.

Reply:
WooCommerce  suggests using `class_exists( 'woocommerce' )`: https://woocommerce.com/document/query-whether-woocommerce-is-activated/

Reply:
I'm happy with any kind of check like this and there are many things that should be stable to check. Works for me 👍